docs/index.md

```diff
@@ -44,11 +44,12 @@ PrivateKey privateKey;
 
 // Values are from the API Integration configuration.
 AdobeClientCredentials org = new AdobeClientCredentials("Org Id", "Technical Account Id", "API Key", "Client Secret", privateKey);
-String token = org.authenticate(org);
+IdentityManagementApi imApi = new IdentityManagementApiImpl("Base URL");
```

Comment:
You can actually remove the `Base URL` now. I just pushed an update to provide no-args constructor to use the default endpoint.